angular/core/services/intercom_service.coffee

```diff
@@ -5,7 +5,7 @@ angular.module('loomioApp').factory 'IntercomService', ($rootScope, $window, App
     key: group.key
     name: group.name
     description: group.description.substring(0, 250)
-    admin_link: AppConfig.baseUrl+"/admin/groups/#{group.key}"
+    admin_link: AppConfig.baseUrl+"admin/groups/#{group.key}"
```

Comment:
@rdbartlett thanks for the fix, but I think it's better to fix it in AppConfig.baseURL.. seems to me like we'd want any url or piece of url to make sense by itself.